src/analysis/retail/rogue/outlaw/modules/core/FinisherUse.tsx

```diff
@@ -0,0 +1,82 @@
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { CastEvent } from 'parser/core/Events';
+import { BadColor, GoodColor } from 'interface/guide';
+import { ResourceLink} from 'interface';
+import DonutChart from 'parser/ui/DonutChart';
+import Statistic from 'parser/ui/Statistic';
+import { STATISTIC_ORDER } from 'parser/ui/StatisticBox';
+import RESOURCE_TYPES from 'game/RESOURCE_TYPES';
+import getResourceSpent from 'parser/core/getResourceSpent';
+import {
+  FINISHERS,
+  getMaxComboPoints,
+} from '../../constants';
+import Finishers from '../features/Finishers';
+
+export default class FinisherUse extends Analyzer {
+  static dependencies = {
+    finishers: Finishers,
+  };
+
+  protected finishers!: Finishers;
+
+  totalFinisherCasts = 0;
+  lowCpFinisherCasts = 0;
+
+  constructor(options: Options) {
+    super(options);
+    this.addEventListener(Events.cast.by(SELECTED_PLAYER).spell(FINISHERS), this.onCast);
+  }
+
+  get maxCpFinishers() {
```

Comment:
I'm unaware of Outlaw's use of Animacharged; is that something that's worth representing here? You can check the Assassination module for an implementation if it is.

This also isn't an issue for this PR.

Reply:
I chose to not really bother with the keep it rolling - resounding clarity build for now as it has fallen quite a bit behind now, just checked the top 100 of every boss and couldnt find a single log playing RC.
It's something I want to support in the future of course but for now it seemed irrelevant.